flink/src/main/java/org/apache/iceberg/flink/FlinkConfigOptions.java

```diff
@@ -23,9 +23,9 @@
 import org.apache.flink.configuration.ConfigOption;
 import org.apache.flink.configuration.ConfigOptions;
 
-public class FlinkTableOptions {
+public class FlinkConfigOptions {
```

Comment:
This looks like an existing public API, renaming might have some challenges or at the very least need a comment in the migration guide. Another option would be to extend FlinkConfigOptions into a deprecated FlinkTableOptions so as not to break compatiblity.

Reply:
the class is use internally. the config keys are public contract and weren't be changed. Let me move this one out to a separate small PR

Reply:
https://github.com/apache/iceberg/pull/2742